src/fr/articles/landmarks-aria/index.njk

```diff
@@ -2,136 +2,207 @@
-    …
-  &lt;/div&gt;
-&lt;div role="main"&gt;
-&lt;div role="navigation" aria-label="Fil d’Ariane"&gt;[…]&lt;/nav&gt;
-  &lt;div role="region"&gt;
-    &lt;div role="header"&gt;
-      &lt;h1&gt;Actualités&lt;/h1&gt;
-      &lt;p&gt; 21 juin 2018 - Fête de la musique&lt;/p&gt;
-    &lt;/header&gt;
-    &lt;p&gt;contenu de la région&lt;/p&gt;
-    &lt;div role="contentinfo"&gt;
-      &lt;p&gt;contenu du pied de région&nbsp;: copyright…&lt;/p&gt;
-    &lt;/div&gt;
-  &lt;/div&gt;
-  &lt;div role="article"&gt;
-  […]
-  &lt;/div&gt;
-
-[…]
-
-&lt;/div&gt;
-&lt;div role="complementary"&gt;
-  &lt;p&gt;contenu annexe en relation avec le contenu principal&lt;/p&gt;
-    […]
-  &lt;/div&gt;
-&lt;div role="contentinfo"&gt;
-  &lt;p&gt;contenu du pied de page&nbsp;: copyright…&lt;/p&gt;
-    […]
-&lt;/div&gt;
-&lt;/body&gt;
-</code></pre>
-
-
-<h3>Usages</h3>
-<ul>
-  <li>Il ne peut y avoir qu’une seule balise <code>main</code> ou donc un seul <code>role=main</code> par page afin d’identifier le contenu principal d’une page.</li>
-  <li>La balise <code>section</code> ou <code>role=region</code> permet de thématiquement regrouper du contenu, par exemple, des chapitres d'un livre, les grandes parties d'une page d'accueil...</li>
-  <li>La balise <code>article</code> ou <code>role=article</code> est une portion autonome de la page, en général, réutilisable, par exemple, un article de blog, d'un journal, un commentaire d'un blog...</li>
-  <li>Le <code>role=navigation</code> ou la balise <code>nav</code> permet d’identifier les principales navigations internes au site. Les bonnes pratiques demandent que l’on utilise 3 balises ou rôles <code>nav</code>ou navigation maximum par page.</li>
-  <li>La balise <code>footer</code> ou le <code>role=contentinfo</code> permettent d’identifier le pied de page. Il doit être unique dans la page, sauf lorsqu'il est présent dans des balises <code>section</code>, <code>role=region</code> ou <code>article</code>, <code>role=article</code>.</li>
-  <li>La balise <code>header</code> ou <code>role=header</code> identifie l’en-tête de la page. Lui aussi doit être unique, sauf lorsqu'il est présent dans des balises <code>section</code>, <code>role=region</code> ou <code>article</code>, <code>role=article</code>.</li>
-  <li>Pour identifier le formulaire de recherche dans le site, on utilisera seul le <code>role=search</code> <abbr>ARIA</abbr>.</li>
-  <li>Si besoin, le contenu annexe au contenu principal de la page (information complémentaire, publicité…) doit être identifié en utilisant la balise <code>aside</code> ou le <code>role=complementary</code>.</li>
+<p>Pour ajouter de la sémantique HTML au contenu d'une page, 2 possibilités :</p>
+<ol>
+  <li>
+    <p><u>Pour une page <em>en HTML5</em></u> (recommandé) : utiliser les <em>balises sémantiques </em>(<code>main</code>, <code>header</code>, <code>footer</code>, <code>aside</code>, ...) qui y ont été introduites.
```

Comment:
To stay consistent with the rest of the website, underlined text is for links only.